### PR TITLE
prepare 0.10.1

### PR DESCRIPTION
**Pull request by blizzz**

### CODE REVIEW

**Comment by blizzz** [comment]
addresses #394 

@marcelklehr if prepared a draft in the releases, if you want to have a look

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	--exclude=/.scrutinizer.yml \
+	--exclude=/.travis.yml \
+	--exclude=/Makefile \
+	$(project_dir)/ $(sign_dir)/$(app_name)
+	tar -czf $(build_dir)/$(app_name)-$(version).tar.gz \
+		-C $(sign_dir) $(app_name)
+	@if [ -f $(cert_dir)/$(app_name).key ]; then \
+		echo "Signing package…"; \
+		openssl dgst -sha512 -sign $(cert_dir)/$(app_name).key $(build_dir)/$(app_name)-$(version).tar.gz | openssl base64; \
+	fi
```
Comment (marcelklehr) [inline Makefile]: :+1: for the makefile :)

**Comment by marcelklehr** [comment]
Release draft looks good!

**Comment by blizzz** [comment]
released 🎉 🍰

**Pull request merged**